### PR TITLE
Add missing ts() function

### PR DESCRIPTION
**Pull request by scardinius**
Hi,

I'm going to translate whole extension into Polish. Therefore I added missing ts() function.

By the way, do you know more user friendly solution for communition of extension dependency?
[campaigngoodies.php:36](https://bitbucket.org/caltha/pl.infozmiana.campaigngoodies/src/25574a8bb38c56f46f048981dd9e12447f9e3bdb/campaigngoodies.php?fileviewer=file-view-default#campaigngoodies.php-36)

### CODE REVIEW

**Comment by bjendres** [comment]
Hi. Great, thank you!

Could I ask you, possibly, to please do it "properly"? I know, I used the same (bad) approach in CiviSEPA, but the correct implementation would be the one you can see in https://github.com/systopia/de.systopia.householdmerge:
- use `{ts}string{/ts}` instead of `{{ts('string')}}` in tpl files
- the `l10n` folder contains the localisation for the extension's domain
- the `ts()` and `{ts}{/ts}` functions get a `domain` attribute

This way it'll be much easier to install the localisation as well, as it comes right with the extension code.

File: partials/campaign_copy.html [hunk 1]
```diff
@@ -1,31 +1,31 @@
 <div id="campaign_container">
    <div id="campaign_summary">
-      <h1>Clone "{{currentCampaign.title}}"</h1>
+      <h1>{{ts('Clone', {domain: 'de.systopia.campaign'})}} "{{currentCampaign.title}}"</h1>
```
Comment (scardinius) [inline partials/campaign_copy.html:3]: @systopia  syntax `{ts}string{/ts}` doen't work in such angular templates. 
This version is ok?

File: partials/campaign_dashboard.html [hunk 2]
```diff
@@ -19,70 +19,70 @@
...
       </div>
       <div id="info">
          <table class="cmpinfo">
 
             <tr>
-              <td>{{ts('Status')}}</td>
-              <td>{{currentCampaign.status_id_text ? currentCampaign.status_id_text : 'not set' }}</td>
-              <td><a crm-confirm="{message: ts('Are you sure you want to apply this status id to all subcampaigns?')}" on-yes="applyToChildren('status_id', currentCampaign.status_id)" class="button">{{ts('Apply to Subcampaigns')}}</a></td>
+              <td>{{ts('Status', {domain: 'de.systopia.campaign'})}}</td>
+              <td>{{currentCampaign.status_id_text ? currentCampaign.status_id_text : ts('not set') }}</td>
```
Comment (scardinius) [inline partials/campaign_dashboard.html]: @systopia How to add domain key to `ts('not set')`?

Comment (bjendres) [inline partials/campaign_dashboard.html]: see e.g. here: https://github.com/systopia/de.systopia.householdmerge/blob/master/CRM/Householdmerge/Form/Task/Merge.php#L68

File: partials/campaign_copy.html (hunk 1 above)
Comment (bjendres) [inline partials/campaign_copy.html:3]: Yes, sorry, I didn't see the angular there. If that works for you it's fine!

**Comment by scardinius** [comment]
And now is it ok?

**Comment by bjendres** [comment]
> And now is it ok?

Sorry, I have no means of testing it right now. If you feel confident, I'll merge it and we can fix it later if it turns out to be broken...

**Comment by michau** [comment]
I guess merging it would be most convenient - we'll get to translating the extension very soon, so it will be additionally tested by us - we'll submit additional PRs, if we find anything.

**Comment by bjendres** [comment]
@nikobo: Could you merge this please?

**Pull request merged**